mitosheet/mitosheet/public/v3/formatting.py

```diff
@@ -16,11 +26,12 @@ def add_conditional_formats(
     for conditional_format in conditional_formats:
         for filter in conditional_format.get('filters'):
             # Start with the greater than condition
-            if filter['condition'] != 'greater':
+            operator = operator_map[filter['condition']]
```

Comment:
This will error if the condition isn't in the map - e.g. non-number conditions. We should us a .get on the operator_map.

Reply:
Done!